lib/rubygems/gemcutter_utilities.rb

```diff
@@ -275,6 +275,9 @@ def wait_for_otp(webauthn_url)
     url_with_port = "#{webauthn_url}?port=#{port}"
     say "You have enabled multi-factor authentication. Please visit #{url_with_port} to authenticate via security device. If you can't verify using WebAuthn but have OTP enabled, you can re-run the gem signin command with the `--otp [your_code]` option."
 
+    say "\n[WARNING] It looks like your default browser is Safari. Due to limitations within Safari, you will be unable to " \
```

Comment:
Style nit: I think this would be easier to read as a conventional `if/end` block.